product_variant_inactive/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Module name
+===========
+
+This module was written to extend the functionality of Product Variants
+and allow you to make activate or deactivate products using the two buttons
+
+.. image:: img/img_01.png
```

Comment:
For screenshot please trim to the necessary information (no need browser title or your desktop)